lib/typescript/render/providers/facebook/facebookModel.ts

```diff
@@ -3,9 +3,45 @@ export interface Attachment {
 }
 export interface SimpleAttachment {
   type: 'image' | 'video' | 'audio' | 'file' | 'fallback';
-  payload: {
-    title?: string;
-    url?: string;
+  title?: string;
+  url?: string;
+  payload?: {title?: string; url?: string} | null;
+}
+
+export interface URLButton extends Content {
+  type: 'web_url';
+  url: string;
+  title: string;
+}
+
+export interface PostbackButton extends Content {
+  type: 'postback';
+  title: string;
+  payload: string;
+}
+
+export interface CallButton extends Content {
+  type: 'phone_number';
+  title: string;
+  payload: string;
+}
+
+export interface LoginButton extends Content {
+  type: 'account_link';
+  url: string;
+}
+
+export interface LogoutButton extends Content {
+  type: 'account_unlink';
+}
+
+export interface GamePlayButton extends Content {
+  type: 'game_play';
+  title: 'Play';
+  payload?: string;
+  game_metadata?: {
+    player_id?: string;
+    context_id?: string;
   };
 }
```

Comment:
Refactored the typings of the Buttons + added more types 

- https://developers.facebook.com/docs/messenger-platform/reference/buttons